config/initializers/resque.rb

```diff
@@ -28,3 +28,11 @@
 # require 'resque/failure/redis'
 # Resque::Failure::MultipleWithRetrySuppression.classes = [Resque::Failure::Redis]
 # Resque::Failure.backend = Resque::Failure::MultipleWithRetrySuppression
+
+# logs Resque failures and sends them to ExceptionNotification
+require 'resque/failure/multiple'
+require 'resque/failure/redis'
+require 'exception_notification/resque'
+
+Resque::Failure::Multiple.classes = [Resque::Failure::Redis, ExceptionNotification::Resque]
```

Comment:
ExceptionNotification's EmailNotifier tries to use deliver_now if it's available (so shouldn't trigger a background job, so shouldn't cause infinite loop issues here)

https://github.com/smartinez87/exception_notification/blob/60e158888bbff96f7a9ae00ae97305cdd33726e9/lib/exception_notifier/email_notifier.rb#L199

I think we're good!